app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -3722,44 +3726,12 @@ class BrowserTabViewModelTest {
     }
 
     @Test
-    fun whenDownloadIsCalledThenDownloadRequestStartedPixelFired() {
+    fun whenDownloadIsCalledThenDownloadRequestedForUrl() = runTest {
         val pendingFileDownload = buildPendingDownload(url = "http://www.example.com/download.pdf", contentDisposition = null, mimeType = null)
 
         testee.download(pendingFileDownload)
 
-        verify(mockPixel).fire(AppPixelName.DOWNLOAD_REQUEST_STARTED)
-    }
-
-    @Test
-    fun whenDownloadIsCalledForDataAndFinishedThenDownloadRequestStartedAndDownloadRequestSucceededPixelsFired() {
```

Comment:
These tests were removed from `BrowserTabViewModelTest` as the code was moved to `FileDownloadCallback`.